src/core/namespace.c

```diff
@@ -78,6 +78,14 @@ typedef enum MountMode {
         _MOUNT_MODE_MAX,
 } MountMode;
 
+typedef enum MountEntryState {
+      MOUNT_UNAPPLIED,
```

Comment:
Maybe call this `MOUNT_PENDING`?

Reply:
yeah, MOUNT_PENDING sounds better to me, too. but also don#t care too much, set the (weak) green label